### ISSUE TITLE
Validation Comment Drop Down Menu

### ISSUE THREAD

**Issue by devonsnyder** [ProjectSidewalk/SidewalkWebpage#3113]
##### Brief description of problem/feature
Creation of a drop-down menu for the validation missions so users can select common comments that apply to specific label types. There could also be an option for "Other" where the user can fill in their own comment. Drop-down comments could be pulled/based off this document that was created when we were all validating Yochai's students last year: https://docs.google.com/document/d/1pxpuDLuidE4aponNKh89OksSqwilA7fxttC2tx4AMF8/edit

**Comment by misaugstad** [comment]
Agreed! We also talked about wanting to have something like this before we tried to add the ability to add comments on mobile in [#3055](https://github.com/ProjectSidewalk/SidewalkWebpage/issues/3055#issuecomment-1317431151)!

**Comment by jonfroehlich** [comment]
Strongly agree. There could also be some more clever ways of handling this on the UI side than a dropdown (which requires two clicks, doesn't show visible choices, etc.)

**Comment by jonfroehlich** [comment]
I think the options could also be based on: (1) my now nearly 15k validations on Seattle and Chicago and an analysis of common errors; (2) our prior work analyzing common errors; (3) an examination of the Description field for why users disagree.

**Comment by jonfroehlich** [comment]
As we're talking about this a bit in https://github.com/ProjectSidewalk/SidewalkWebpage/issues/3096, I wanted to add my thoughts here. 

When the user hits "disagree", I think it would be nice if we had a lightweight way of selecting some common reasons for the mistake (which differ based on each label type). I want us to be creative about this to allow the user to give us more info without slowing down validations too much--a difficult balance. For example, you could imagine "mousing over" the Disagree button and it pops up some common mistakes to select and then you hit Disagree to submit...

**Comment by misaugstad** [comment]
I like that a lot!

**Comment by yeisenberg** [comment]
Talked about this issue today and re-voting for validation drop down...thanks!

**Comment by misaugstad** [comment]
We also talked about this as a way to remove the need for moderation of validation comments, since all feedback between users will be closed-form! We can always include an open-form "other" option that is never shown to other users if we want. We can use that data to inform additional validation comment options.

**Comment by misaugstad** [comment]
We have buttons with reasons to disagree or be unsure in the new version of the Validate page. However, we plan to keep around the current Validate page since it is faster. We should probably implement something like a drop-down menu with disagree/unsure reasons for the regular Validate page now. We can use the same reasons we already have for the other.
